### DISCUSSION TITLE
When will there be a new version of Skyline?

### DISCUSSION THREAD

**Discussion by Franto011** [skyline-emu/skyline#1582] [Q&A] [answered]
Hello, I understand that you will release version 0.0.4 of Skyline, could you give us a tentative date?

If this rumor is not true, it would also be very kind of you to deny it.

I have high hopes for the emulator and can't wait to try new versions.

Thank you very much for your great work!

**Comment by Crytonics** [comment, marked as answer]
We dont use versions (x.x.4...), we use builds (CI #1200...). You can find them in tab "Actions" on github.

**Reply by Franto011** [reply]
I know but I understood that they have not made any more updates or at least publications because it was moving to version 0.0.4

**Reply by Crytonics** [reply]
In the "Actions" there is updates to skyline you can download them. We just dont use versions. Insead of 0.0.4... we use builds CI 1200.... Each build is update to skyline.

**Reply by Crytonics** [reply]
Here https://github.com/skyline-emu/skyline/actions/runs/2479814673

**Reply by Franto011** [reply]
Are you part of the developers? if so, is there any special reason why there is no update these days?

**Reply by Crytonics** [reply]
I am not dev, but i think that Skyline will jump to 1.0 when hits play store but i am not sure. For now we will update Skyline with builds

**Reply by Franto011** [reply]
Well then, since you are so sure that there will be no version 0.0.4 haha ​​better, let's hope if someone clarifies doubts, it's been a week without any updates and it's something quite peculiar, since no updates have been pronounced on the matter if perhaps there will no longer be updates so often.

**Reply by Crytonics** [reply]
Well last update was 6 days ago, devs can't every day update skyline. Then need somethings pause and they are working hard. I told you there were 200+ updates in the last 2 month so there was a ton updates.

**Reply by Franto011** [reply]
> Bueno, la última actualización fue hace 6 días, los desarrolladores no pueden actualizar el horizonte todos los días. Entonces necesitan una pausa y están trabajando duro. Te dije que hubo más de 200 actualizaciones en los últimos 2 meses, así que hubo un montón de actualizaciones.

Well, you're right about that, great developers.
